### PR TITLE
Add 2.1.0 to 3.0.0 migration doc

### PR DESCRIPTION
**Pull request by npamudika**
Fixes related to https://github.com/wso2/docs-apim/issues/434

### CODE REVIEW

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 1]
```diff
@@ -1,303 +1,1878 @@
...
 
-`/sequences/_resource_mismatch_handler_.xml`
+    !!! warning
+        When moving the Synapse configurations, **do not replace** the following set of files as they contain some modificatiosn in API-M 3.0.0 version.
 
-`/sequences/_sandbox_key_error_.xml`
+        -   /api/_RevokeAPI_.xml
+        -   /sequences/_cors_request_handler_.xml
+        -   /sequences/main.xml
+        -   proxy-services/WorkflowCallbackService.xml
```
Comment (vithu30) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: In 2.1.0 to 2.6.0 migration doc [1], there are list of sequences that should not be replaced in the migrated version. Did we check on that?

[1] https://docs.wso2.com/display/AM260/Upgrading+from+the+Previous+Release#210

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 2]
```diff
@@ -1,303 +1,1878 @@
...
-`/sequences/_production_key_error_.xml`
+6.  Move all your Synapse configurations to API-M 3.0.0 pack.
+    -   Move your Synapse super tenant configurations.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/deployment/server/synapse-configs/default` directory with the copied contents.
+    -   Move all your tenant Synapse configurations.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/tenants` directory with the copied contents.
 
-`/sequences/_resource_mismatch_handler_.xml`
+    !!! warning
+        When moving the Synapse configurations, **do not replace** the following set of files as they contain some modificatiosn in API-M 3.0.0 version.
```
Comment (vithu30) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Typo in `modificatiosn`

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 3]
```diff
@@ -1,303 +1,1908 @@
...
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('AbnormalResponseTime', 'publisher');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('AbnormalBackendTime', 'publisher');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('AbnormalRequestsPerMin', 'subscriber');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('AbnormalRequestPattern', 'subscriber');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('UnusualIPAccess', 'subscriber');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('FrequentTierLimitHitting', 'subscriber');
+        INSERT INTO AM_ALERT_TYPES (ALERT_TYPE_NAME, STAKE_HOLDER) VALUES ('ApiHealthMonitor', 'publisher');
+
+        CREATE TABLE IF NOT EXISTS AM_APPLICATION_ATTRIBUTES (
+        APPLICATION_ID int(11) NOT NULL,
```
Comment (vithu30) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Shall we format the DB scripts?

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md (hunk 1 above)
Comment (npamudika) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Yes. There is a discussion happened internally and based on the conclusion, edited the list.

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md (hunk 2 above)
Comment (npamudika) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Fixed

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md (hunk 3 above)
Comment (npamudika) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Done

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 4]
```diff
@@ -1,303 +1,1908 @@
...
 
+7.  Move all your Execution plans from `<API-M_2.6.0_HOME>/repository/deployment/server/executionplans` directory to `<API-M_3.0.0_HOME>/repository/deployment/server/executionplans` directory.
 
-6.  Move all your tenant Synapse configurations by copying the contents in the `<CURRENT_API-M_HOME>/repository/tenants` directory and adding the copied contents to the `<API-M_2.6.0_HOME>/repository/tenants` directory.
+    !!! note
+        If you are working with a **clustered/distributed API Manager setup**, follow this step on the **Traffic Manager** node.
 
-        !!! warning
-    **NOTE:** Copy the files listed below from the `<API-M_2.6.0_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory and replace the corresponding files in the `<API-M_2.6.0_HOME>/repository/tenants/<tenant-id>/synapse-configs/default/sequences` directory.
+8.  If you manually added any custom OSGI bundles to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory.
```
Comment (vithu30) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Change the API-M version as 2.1.0 in the path 
``` suggestion
8.  If you manually added any custom OSGI bundles to the `<API-M_2.1.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
```

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 5]
```diff
@@ -1,303 +1,1908 @@
...
-6.  Move all your tenant Synapse configurations by copying the contents in the `<CURRENT_API-M_HOME>/repository/tenants` directory and adding the copied contents to the `<API-M_2.6.0_HOME>/repository/tenants` directory.
+    !!! note
+        If you are working with a **clustered/distributed API Manager setup**, follow this step on the **Traffic Manager** node.
 
-        !!! warning
-    **NOTE:** Copy the files listed below from the `<API-M_2.6.0_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory and replace the corresponding files in the `<API-M_2.6.0_HOME>/repository/tenants/<tenant-id>/synapse-configs/default/sequences` directory.
+8.  If you manually added any custom OSGI bundles to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
 
-`_auth_failure_handler_.xml`
+9.  If you manually added any JAR files to the `<API-M_3.0.0_HOME>/repository/components/lib` directory, copy those and paste them in the `<API-M_3.0.0_HOME>/repository/components/lib` directory.
```
Comment (vithu30) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Change the API-M version:
``` suggestion
9.  If you manually added any JAR files to the `<API-M_2.1.0_HOME>/repository/components/lib` directory, copy those and paste them in the `<API-M_3.0.0_HOME>/repository/components/lib` directory.
```

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md (hunk 4 above)
Comment (npamudika) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Will fix in the next PR

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md (hunk 5 above)
Comment (npamudika) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Will fix in the next PR

**Pull request merged**